index.js

```diff
@@ -37,4 +47,4 @@ const stringify = (object, type) => {
     }
 }
 
-module.exports = { parse, stringify }
+module.exports = { parse, stringify, removeFileExt }
```

Comment:
But the `input..cson` files will be detected as the type `.cson` instead of just `cson` so it should work.

In any case, I don't think this really needs to be tested. The function is one line, and super simple at that.